cmd/provider/main.go

```diff
@@ -122,8 +122,8 @@ func main() {
 	}
 
 	if *enableManagementPolicies {
-		o.Features.Enable(features.EnableAlphaManagementPolicies)
-		log.Info("Alpha feature enabled", "flag", features.EnableAlphaManagementPolicies)
+		o.Features.Enable(feature.EnableBetaManagementPolicies)
+		log.Info("Beta Management Policies feature enabled", "flag", feature.EnableBetaManagementPolicies)
```

Comment:
`features.EnableAlphaManagementPolicies` is a provider feature flag that does not come with the runtime. Changing this wont work.

Reply:
oh i see, i guess i'll go back to the drawing board and see if i can get this provided updated to 1.15.x so we can use the beta feature flag.